TestCases/FireStoreCollection.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
```

Comment:
I added MockDataSetup.cs in my branch.
Steps to setup mock data and firestore for testing:
1. Remove Skip in fact attribute, so that test can be executed.
2. Run the all tests inMockDataSetUp.
3. Add skip again to fact attribute.
4. OrderBy test cases require firestore indexes to be created.
5. You can check the error message in test explorer, and you can get a link there itself to create a firestore index.
6. Rerun all the tests again.

Thanks for your contribution :)